course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,103 @@
+
+# Lecture schedule
+| Week | Date   | Topic                                               | Instructor     |
+|------|--------|-----------------------------------------------------|----------------|
+| 1    | Sep 07 | Welcome to the course! Intro to programming         | Everyone       |
+| 2    | Sep 12 | Basic R – vectors, data frames                      | Joel           |
+| 3    | Sep 14 | Basic R – scripting, conditionals, loops            | Joel           |
+| 4    | Sep 19 | Intro to dplyr                                      | Joel           |
+| 5    | Sep 21 | More dplyr                                          | Joel           |
+| 6    | Sep 26 | Qualitative review of population models             | Madeleine      |
+| 7    | Sep 28 | Qualitative population models in 2 dimensions       | Madeleine      |
+| 8    | Oct 03 | Fixed points, null clines, and stability            | Madeleine      |
+| 9    | Oct 05 | Modeling in R                                       | Madeleine      |
+| 10   | Oct 10 | Fitting models to data                              | Madeleine      |
+| 11   | Oct 12 | Linear regression population dynamics models?       | Luke           |
+| 12   | Oct 17 | Introduce data sets, hypotheses, start projects     | Luke           |
+| 13   | Oct 19 | Project work, prodigenR, tidy data, git via rstudio | Luke           |
+| 14   | Oct 24 | Data visualization in R                             | Lindsay        |
+| 15   | Oct 26 | Project work, data visualizations                   | Lindsay        |
+| 16   | Oct 31 | Reproducible workflow, metadata, Rmarkdown          | Lina           |
+| 17   | Nov 02 | Project work, metadata                              | 1 TA/group     |
+| -    | Nov 07 | Fall break                                          | -              |
+| -    | Nov 09 | Fall break                                          | -              |
+| 18   | Nov 14 | Scientific collaboration, Git and GitHub            | Elliot         |
+| 19   | Nov 16 | Project work                                        | 1 TA/group     |
+| 20   | Nov 21 | Preparing a scientific item for academic publishing | Lina           |
+| 21   | Nov 23 | Project work                                        | 1 TA/group     |
+| 22   | Nov 28 | Project work                                        | 1 TA/group     |
+| 23   | Nov 30 | Project work                                        | Everyone       |
+| 24   | Dec 05 | Project work                                        | Everyone       |
+
+# Assessment schedule 
+| Assesment             | Type                | Due date   | Marks |
+|-----------------------|---------------------|------------|-------|
+| Getting set up        | Individual          | Sep 11     | 4     |
```

Comment:
@mbonsma @QuLogic and I talked briefly about this today. We agree that the due date should be on the same day that we have office hours so that student's can come and ask last minute questions.

We largely agreed that it would be better if students had 8 days rather than 3 days (excluding weekends) to hand in the assignment, especially since around 10 days seems to be standard and we don't want to scare students away by making them work on the weekends right away... The drawback of this is that it would be harder to follow up promptly if there are particular concepts that many students don't understand. We could remedy this with a socrates online quiz at the beginning of the following lecture (maybe during the break?), and repeat any concept that are unclear for many students. We just don't want to add to much overhead...